### PR TITLE
FCCeeBaselineDet

### PR DESCRIPTION
**Pull request by nalipour**
Addresses # .

Changes proposed in this pull-request:

- A detector for FCCee working fully within the FCCSW

To be done:

- [ ]
- [ ]

### CODE REVIEW

File: Detector/DetFCCeeBaseline1/compact/materials.xml [hunk 1]
```diff
@@ -0,0 +1,235 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jlingema) [inline Detector/DetFCCeeBaseline1/compact/materials.xml]: I guess this can be deleted and missing materials be added to `DetCommon/compact/materials.xml`?

**Comment by nalipour** [comment]
Dear Joschka,

Thanks for your comments. I have renamed all the files by removing the versions as you have asked. Although this geometry still does not follow the folder structures as it is shown in the documentation because it is a quite preliminary one and many changes are to be implemented. It is just a first version fully working. As soon as we can freeze a geometry I will implement the folder structure as asked

Cheers,
Nilou

**Comment by jlingema** [comment]
Hi Nilou,

Thanks for the updates. One remaining question: This supersedes #187, correct?

Cheers,
Joschka

ok to test

**Comment by nalipour** [comment]
Hi Joschka,

Yes, this will replace #187 

Cheers,
Nilou

**Comment by jlingema** [comment]
Hi Nilou, thanks for clarifying, I closed the other PR. 

I had another look and it seems that there is still `Detector/DetFCCeeBaseline1/compact/Materials.xml`, can we merge that also into `Detector/DetCommon/compact/materials.xml`?

Additionally, the master file still tries to include the files with the version in the name (which is why the test failed).
Cheers, Joschka

**Comment by nalipour** [comment]
Hi Joschka,

For the Materials.xml, do you really want everything in DetCommon? I think they are specific to the CLIC detector.

Yes, I am working for the test. There were few bugs. I will commit them right now.

Thanks,
Nilou

**Comment by jlingema** [comment]
Hi Nilou, yes, please move all into one `materials.xml` that way others can re-use and don't have to browse through multiple files (and at the moment the file is still relatively small). Just have a look that the materials which you add are actually used.
Cheers, Joschka

**Comment by nalipour** [comment]
Hi Joschka,

Sorry to ask, but could you tell me where I can see the output of your tests?
On my local version, it succeeds.

Cheers,
Nilou

**Comment by jlingema** [comment]
Hi,
clicking on the little red cross next to the commit id will get you to the Jenkins page, where you'll have to click on the link to the build (default, next to the red blob at the bottom) and then on the left-hand panel there is a link to the parsed and [full console output](https://phsft-jenkins.cern.ch/job/FCCSW-pullrequests/399/BUILDTYPE=Release,COMPILER=gcc49,LABEL=slc6/consoleFull) -- looks like the overlap check fails.
Cheers,
Joschka

**Comment by nalipour** [comment]
Hi Joschka,

I have removed the overlay test so that the tests run. Warnings are put on the geometry to say that it contains overlaps. For the moment we proceed with a geometry with overlaps since the simulation runs. They will be resolved once we have a more stable geometry.

Thanks,
Nilou

**Comment by jlingema** [comment]
Hi Nilou,

If we get rid of the overlap check as a test, can you please add one that runs the simulation to verify that the xmls are consistent? 

Thanks,
Joschka

**Comment by vvolkl** [comment]
Hi @nalipour,

If you are willing to reset your branch, I think the changes in `Generation` might even merit their own pull request!

**Comment by nalipour** [comment]
Hi Valentin,

Indeed I will make a new branch for that. 
But my previous commits on the geometry are going to be merged at some point?

Thanks,
Nilou

**Comment by zaborowska** [comment]
Hi @nalipour, as I understand, you are working on the new implementation of the detectors and this PR is actually not needed and can be closed?

**Comment by nalipour** [comment]
Hi @zaborowska! 
Yes, you can close the PR at the moment. I might later re-introduce the code.
Cheers,
Nilou

**Comment by zaborowska** [comment]
ok, thanks Nilou!

**Pull request closed without merging**